src/components/Sheetmusic.vue

```diff
@@ -0,0 +1,67 @@
+<script lang='ts'>
+import Vue from 'vue'
+import AsyncComputed from 'vue-async-computed'
+import { OpenSheetMusicDisplay } from 'opensheetmusicdisplay'
+import logger from '../services/logger.js'
+import parsePath from 'path-parse'
+
+Vue.use(AsyncComputed)
+
+export default {
+    name: 'Sheetmusic',
+
+    computed: {
+        name() {
+            return parsePath(this.basename).name
+        },
+        ext() {
+            return parsePath(this.basename).ext
+        },
+        isMusicXml() {
+            return (this.ext === '.mxl' || this.ext === '.musicxml');
+        }
+    },
+    mounted() {
+        if (this.isMusicXml) {
+            const openSheetMusicDisplay = new OpenSheetMusicDisplay(this.$refs.canvas, {
+                autoResize: true,
+                darkMode: true
```

Comment:
https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/blob/79a356bc9c4c09945456920fdd7b3d2474094886/src/OpenSheetMusicDisplay/OpenSheetMusicDisplay.ts#L557-L563

Please use the current user settings to decide whether to set dark mode to true or not.

```ts
export function isDarkModeEnabled() {
	const darkModePreference = window?.matchMedia?.('(prefers-color-scheme: dark)')?.matches
	const darkModeSetting = document.body.getAttribute('data-themes')?.includes('dark')
	return darkModeSetting || darkModePreference || false
}
```

Reply:
I double checked with dark and light theme and the background of the viewer is always black. So `true` is kinda correct here. Otherwise I could alter the background of the viewer if that makes sense and is doable

Reply:
Ah, yes, you're right. I completely forgot!
We do force the dark theme on viewer 😅🙈
Can you then only add a small comment to explain why it's hard coded to true ?

Reply:
Haha, will do :)